CommandLine/Accept.cs

```diff
@@ -86,6 +86,29 @@ public static class Accept
                 return null;
             }));
 
+        public static ArgumentsRule LegalFilePathsOnly(
```

Comment:
I'm going to leave the name as-is for now. I have a list of a few methods I'd like to rename and it might be best to do them all at once.